qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -391,249 +344,127 @@ def error_message(self):
 
         return self._api_error_msg
 
-    def queue_position(self):
+    def queue_position(self) -> int:
```

Comment:
Maybe a small mishap 😅. But, since `None` could be returned, the the type should be `Optional[int]`